integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java

```diff
@@ -276,12 +277,12 @@ public interface OciConfigBean {
     int imdsTimeoutMilliseconds();
 
     /**
-     * The list of {@link AuthStrategy} names
-     * (excluding {@link AuthStrategy#AUTO}) that
-     * are potentially applicable for use. Here, "potentially applicable or use" means that it is set explicitly by
-     * {@link #authStrategy()}, or else explicitly or implicitly by {@link #authStrategies()}.
+     * The list of {@link AuthStrategy} names (excluding {@link AuthStrategy#AUTO}) that
+     * are potentially applicable for use. Here, "potentially applicable or use" means that it is set using the
+     * {@link #authStrategy()} attribute on this config bean. If not present then the fall-back looks to use the values
+     * explicitly or implicitly set by {@link #authStrategies()}.
      *
-     * @return the list of potential auth strategies that are applicable
+     * @return the list of potential auth strategies that are applicable for use
```

Comment:
no need for "for use"; adds nothing. Maybe just "the list of applicable authentication strategies". Or are they strategy descriptors (the term used in the terminology you pointed the users at originally)?

Reply:
pushed